index.html

```diff
@@ -22,21 +22,34 @@ <h1 class="header-left">
             </a>
         </div>
     </header>
+    
+    <!-- 会社情報 --> 
     <section class="company-info">
         <div class="info-inner two-col">
           <div class="info-title">
             <h2>会社情報</h2>
           </div>
-      
-          <dl class="info-list">
-            <dt>会社名</dt><dd>有限会社 ヤタカ物産</dd>
-            <dt>所在地</dt><dd>〒379-2154 群馬県前橋市天川大島町1371-7</dd>
-            <dt>電話番号</dt><dd>027-263-8009</dd>
-            <dt>代表取締役</dt><dd>矢部 宗雄</dd>
-            <dt>事業内容</dt><dd>建材各種販売・特注家具製造販売</dd>
-          </dl>
+
+          <div class="info-list">
+            <dl class="info-list-detail">
+             <dt>会社名</dt><dd>有限会社 ヤタカ物産</dd>
+             <dt>所在地</dt><dd>〒379-2154 <br>群馬県前橋市天川大島町1371-7</br></dd>
+             <dt>電話番号</dt><dd>027-263-8009</dd>
+             <dt>代表取締役</dt><dd>矢部 宗雄</dd>
+             <dt>事業内容</dt><dd>建材各種販売・特注家具製造販売</dd>
+            </dl>
+          </div>
         </div>
       </section>
+
+<!-- ご相談・お問い合わせ -->
+<section class="contact-cta">
+  <div class="cta-inner">
+    <div class="cta-link">
+      <a href="/contact.html">ご相談・お問い合わせ <img src="/img/arrow.png"></a>
```

Comment:
画像にはalt属性が必須なので、空文字でいいので指定しておくといいかも！
画像の中に文字があればそれを指定、装飾目的ならば今回のように空文字でOKです👍
```suggestion
      <a href="/contact.html">ご相談・お問い合わせ <img src="/img/arrow.png" alt=""></a>
```

Reply:
ご指示ありがとう！alr追記しました🙏